floris/floris_model_utils.py

```diff
@@ -0,0 +1,133 @@
+
+from __future__ import annotations
+
+import os
+from math import ceil
+from typing import (
+    Any,
+    Dict,
+    List,
+    Optional,
+    Tuple,
+)
+
+import numpy as np
+import yaml
+from attrs import define, field
+
+from floris import FlorisModel
+from floris.type_dec import floris_array_converter, NDArrayFloat
+
+
+def nested_get(dic: Dict[str, Any],
```

Comment:
Throughout FLORIS, the style convention for split line arguments is to open the parenthesis, then list the arguments each on their own line, and then close the parenthesis on the final line:
```python
def nested_get(
    d: Dict[str, Any],
    keys: List[str]
) -> Any:
```

Reply:
fixed